tests/telemetry/app.py

```diff
@@ -20,7 +20,7 @@ def metrics_view():
         "test_metric",
         1.0,
     )
-    namespace_metrics = telemetry_metrics_writer._namespace.get()
+    namespace_metrics = telemetry_metrics_writer._namespace._metrics_data
```

Comment:
flush?

Reply:
I didn't want to look into this test further. We could flush but I didn't know if this would break down stream assertions. This has the same behavior of get without exposing that method.